test/Microsoft.AspNetCore.SignalR.Tests/WebSocketsTransportTests.cs

```diff
@@ -62,7 +62,7 @@ public WebSocketsTransportTests(ServerFixture<Startup> serverFixture, ITestOutpu
                 await webSocketsTransport.StartAsync(new Uri(_serverFixture.WebSocketsUrl + "/echo"), channelConnection,
                     TransferMode.Binary, connectionId: string.Empty);
                 connectionToTransport.Out.TryComplete();
-                await webSocketsTransport.Running.OrTimeout();
+                await webSocketsTransport.Running.OrTimeout(TimeSpan.FromSeconds(10));
```

Comment:
The known flaky SSE test failed in AppVeyor. And the timeout is because we wait 5 seconds in WebSockets before cancelling the ReceiveAsync